src/Extension/FluentVersionedExtension.php

```diff
@@ -447,21 +524,53 @@ public static function prepoulateIdsInLocale($locale, $dataObjectClass, $populat
 
     public function updateLocalisationTabColumns(&$summaryColumns)
     {
-        $summaryColumns['IsDraft'] = [
-            'title'    => 'Saved',
+        $summaryColumns['Status'] = [
+            'title' => 'Status',
             'callback' => function (Locale $object) {
-                if ($object && $object->RecordLocale()) {
-                    return $object->RecordLocale()->IsDraft() ? 'Draft' : '';
+                if ($object->RecordLocale()) {
+                    $recordLocale = $object->RecordLocale();
+
+                    if ($recordLocale->IsModified()) {
+                        return 'Modified';
+                    }
+
+                    if ($recordLocale->IsPublished()) {
+                        return 'Published';
+                    }
+
+                    if ($recordLocale->IsDraft()) {
+                        return 'Draft';
+                    }
+
+                    return 'Not localised';
```

Comment:
I tried to find something to critique here but it's already pretty good. :) Good job.

We probably should _t() some of these labels.